### ISSUE TITLE
Windows ink hates me (and AHP)

### ISSUE THREAD

**Issue by dumbeau** [dumbeau/AutoHotPie#49]
Windows ink can eat my ass.  All of it.  But for the meantime, AHP doesn't work with windows ink, so turn it off in your pen software settings.

I need some way to bypass Windows Ink when a pie menu is active.  Or I need the pie menus to work with windows ink, which doesn't seem likely.

**Comment by Michaelknubben** [comment]
While I have always turned Windows Ink off in the past, that's now becoming difficult as more and more software no longer supports Wintab. 
Windows Ink is hot garbage, but it seems like Wintab is on the way out, any chance you've since figured out a way to make this work? Not having the pie menus pop up with my tablet kind of defeats the point for me 😬

**Comment by dumbeau** [comment]
All of a sudden this seems to be working fine with window ink.

However, it can be a little janky only when moving quickly while performing a windows ink specific control.  Like the pie menu may not open in the exact right place if you open the menu immediately after scrolling by 'pen swiping' down a webpage.  Also if you move the mouse very far away from the center, you'll notice the menu may get a little choppy in an app that uses windows ink controls.

**Comment by Michaelknubben** [comment]
Ah, I see the issue is with using the Wacom side-buttons instead. I'm on 1.0.23 alpha, and can't get a menu to spawn when using my wacom RMB.

**Comment by dumbeau** [comment]
Yikes that's definitely a problem then.  You could assign the pen button to 4th or 5th click considering windows ink doesn't do anything special with those.  But I'll look into RMB though.

If you use 4th or 5th click, you can also set the center slice to be an RMB mouse click function.

**Comment by Michaelknubben** [comment]
That would definitely be an option, but I only have two buttons and need both, one as RMB, one as MMB, as both are crucially important in the software I use most (Blender etc)
You might see this yourself (but... given Wacom's drivers _and_ how bad Windows Ink is, who knows), but when you try to use the RMB in AutoHotpie, you just get the dreaded Windows Ink ring, and nothing else.

**Comment by dumbeau** [comment]
Yeah it seems like I can get everything to work MMB.  I cannot figure out the RMB circle thing.

MMB seems to work a lot better, no circles or anything and it opens the menu without any issues.  Using this with a modifier key or 'Pie Enable Key' might be worth looking into for your situation where you like to have pure MMB action at times.
![image](https://user-images.githubusercontent.com/36665762/190167330-5a3cf766-4601-44be-af7d-649a3a12eed3.png)

**Comment by Michaelknubben** [comment]
No, that works for me! RMB is mostly a preference because it's context menu almost everywhere, hence there's a lot of muscle memory there, but key+MMB will do just fine in the mean time!
Thanks for the quick support, and for making this in the first place! As I saw someone else type: I look for this almost yearly, and this is the first time I stumble upon a working solution!

**Comment by TheMystery28** [comment]
There are still no solutions?

There is no way to tweak windows ink so that the mouse position remains consistent?
